c2corg_ui/static/js/maincontroller.js

```diff
@@ -79,13 +80,10 @@ app.MainController.prototype.isPath = function(path) {
     // path = '/'
     return 'home';
   } else if (path === 'topoguide') {
-    // if topoguide, it can be all kinds of documents.
-    // Articles, xreports and outings have their own line in the sidemenu.
-    return location.indexOf('waypoints') > -1 || location.indexOf('routes') > -1 ||
-           location.indexOf('images') > -1 || location.indexOf('areas') > -1 ||
-           location.indexOf('books') > -1;
+    return app.utils.isTopoguide(location.substring(1));
+  } else {
+    return location.indexOf(path) > -1;
   }
-  return location.indexOf(path) > -1;
```

Comment:
Oh yes i do like :P
 But here I really don't remember writing this one, i just deleted the line because make check complained, but you're right ;p